bedrock/contentful/api.py

```diff
@@ -3,6 +3,7 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 from functools import partialmethod
+from typing import Dict, Union
```

Comment:
I'd love to hear about how using this is advantageous. I've heard that using it in a library can be helpful for users of it, but I'm not sure we'll ever get to a point in this code base where this is worth it. I'd love to hear your thoughts though.

Reply:
Yeah, i hear you. Am in a habit of using typing instead of speccing params and return vals in docstrings. 

It's true we're the main/only users of our code, though, so happy to stop and strip em back if you and Rob aren't that fussed